src/styles/card.scss

```diff
@@ -0,0 +1,82 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid $white-90;
+  border-radius: 5px;
+
+  &__img {
+    height: 134px;
+    max-width: 160px;
+    margin: 0 auto 40px;
+    display: block;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $black;
+    margin-bottom: 4px;
+  }
+
+  &__id {
+    font-size: 10px;
+    line-height: 14px;
+    color: $gray;
+    margin-bottom: 16px;
+  }
+
+  &__rate {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    color: $black;
+    text-decoration: none;
+    line-height: 14px;
+    align-self: flex-end;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    .text {
+      font-size: 12px;
+      line-height: 18px;
+      color: $gray;
+    }
+
+    .value {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      color: $black;
+    }
+  }
+
+  &__button {
+    width: 100%;
+    padding: 12px;
+    text-align: center;
+    text-transform: uppercase;
+    border: none;
+    border-radius: 5px;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    color: $white-100;
+    background-color: $blue;
```

Comment:
Let's add transition to make hover effects smooth. Also, would be great to add cursor pointer for button